plugins/woocommerce-admin/client/homescreen/mobile-app-modal/components/JetpackInstallationStepper.tsx

```diff
@@ -50,6 +51,9 @@ export const JetpackInstallationStepper = ( {
 								className="install-jetpack-button"
 								onClick={ () => {
 									setIsWaitingForRedirect( true );
+									recordEvent(
+										'wcadmin_magic_prompt_install_connect_click'
```

Comment:
```suggestion
										'magic_prompt_install_connect_click'
```

The `wcadmin` prefix is not required, otherwise it'll become 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/3747241/190318661-a45f2dc9-1190-4bc8-b01e-8428f208d7fb.png">

Reply:
@rjchow Should apply to all `wcadmin_` prefix in this PR

Reply:
how'd I miss that... 🤦 Thanks!